pkg/rehearse/jobs.go

```diff
@@ -25,6 +31,8 @@ import (
 	"k8s.io/test-infra/prow/pjutil"
 
 	"github.com/openshift/ci-operator-prowgen/pkg/config"
+
+	"k8s.io/api/core/v1"
```

Comment:
This third-party import should be somewhere higher, before the `ci-operator-prowgen`